docs/docker.md

```diff
@@ -0,0 +1,45 @@
+# Activate cloud orchestrator in the remote server
+
+This page describes how to run cloud orchestrator managing docker instances
+containing Host Orchestrator inside.
+
+## Prepare docker image
+
+Please follow the docker part of
+[README.md](https://github.com/google/android-cuttlefish/blob/main/README.md#docker)
+in `google/android-cuttlefish` github repository, and check if the docker image
+`cuttlefish-orchestration` exists.
+
+## Build cloud orchestrator
+
+To build cloud orchestrator, please execute:
+```bash
+/path/to/cloud-android-orchestration/scripts/docker/build.sh
```

Comment:
avoid the `/path/to/cloud-android-orchestration` prefix using  `git clone https://github.com/google/cloud-android-orchestration.git clone && cd cloud-android-orchestration` as the first step, now the reader understand we are within the repository directory.

Reply:
Done.